libbeat/docs/shared-cloudfoundry.asciidoc

```diff
@@ -0,0 +1,128 @@
+[[running-on-cloudfoundry]]
+=== Run {beatname_uc} on Cloud Foundry
+
+ifeval::["{beatname_lc}"=="filebeat"]
+You can use {beatname_uc} on Cloud Foundry to retrieve and ship logs.
+endif::[]
+ifeval::["{beatname_lc}"=="metricbeat"]
+You can use {beatname_uc} on Cloud Foundry to retrieve and ship metrics.
+endif::[]
+
+ifeval::["{release-state}"=="unreleased"]
+
+However, version {version} of {beatname_uc} has not yet been
+released, no build is currently available for this version.
+
+endif::[]
+
+
+[float]
```

Comment:
I would remove this float attribute, so the section appears in the table of contents on the right side of the page.